README.rst

```diff
@@ -100,6 +100,43 @@ Install matchms from Anaconda Cloud with
 
   conda install --channel nlesc --channel bioconda --channel conda-forge matchms
 
+Example
+=======
+
+Below is a small example of using matchms to calculate the Cosine score between mass Spectrums.
+
+.. code-block:: python
+
+    from matchms.importing import load_from_mgf
+    from matchms.filtering import default_filters
+    from matchms.filtering import normalize_intensities
+    from matchms import calculate_scores
+    from matchms.similarity import CosineGreedy
+
+    file = load_from_mgf("tests/pesticides.mgf")
+
+    def my_filter(spectrum):
+        '''Clean and enhance the spectrums with a filter'''
+        spectrum = default_filters(spectrum)
+        spectrum = normalize_intensities(spectrum)
+        return spectrum
+
+    spectrums = [my_filter(spectrum) for spectrum in file]
+
+    similarity_function = CosineGreedy()
```

Comment:
```suggestion
    # calculate similarity using cosine greedy function # see matchms/similarity/CosineGreedy.py
    similarity_function = CosineGreedy()
```

Reply:
Same as before, comment does not explain code better